pkg/cloudprovider/kubevirt/cloud_test.go

```diff
@@ -1,99 +1,99 @@
-package kubevirt
+// package kubevirt
```

Comment:
Uncomment this for the build to pass.
```suggestion
package kubevirt
```